pyscriptjs/src/remote_interpreter.ts

```diff
@@ -171,13 +170,11 @@ export class RemoteInterpreter extends Object {
      * */
     async loadPackage(names: string | string[]): Promise<void> {
         logger.info(`pyodide.loadPackage: ${names.toString()}`);
-        // the new way in v0.22.1 is to pass it as a dict of options i.e.
-        // { messageCallback: logger.info.bind(logger), errorCallback: logger.info.bind(logger) }
-        // but one of our tests tries to use a locally downloaded older version of pyodide
+        // the new way in Pyodode 0.22 and later a locally downloaded older version of pyodide
         // for which the signature of `loadPackage` accepts the above params as args i.e.
         // the call uses `logger.info.bind(logger), logger.info.bind(logger)`.
         const messageCallback = logger.info.bind(logger) as typeof logger.info;
-        if (this.interpreter.version.startsWith('0.22')) {
+        if (Number(this.interpreter.version.split('.')[1]) >= 22) {
```

Comment:
nit: a funny topic here ... JavaScript already understands semver in strings by design:

```js
'0.23.2' > '0.22'; // true
'0.22' > '0.22'; // false
'0.22.1' > '0.22'; // true
'0.23' > '0.22.100001'; // true
```

accordingly .. .do we really need to do this dance that obviously doesn't scale / work moving forward with major versions too?

Reply:
What about `0.23.10` vs `0.23.9`. This is generally what causes trouble when comparing versions lexicographically.

Reply:
The breaking change in the API that this accounts for is in 0.22.0 (the minor-version release), but it does feel odd to me to do string-wise comparison on what are ultimately string-represented numbers. That said, I don't mind doing this in a Javascript-y way. Will make the change.

Reply:
the thing is that once we have a major this code won't work anymore while using `"1.0" > "0.21"` will, so that there's nothing to change from our side in the future if we just use the current version and compare it against `"0.21"` as string.

it's quirky but less maintenance and logic or surprises prone.

Reply:
But you do have the problem that "0.100.0" < "0.22"...

Reply:
with our time to release I don't see that as a real-world issue to consider + I am not a big fan of `0.100` releases ... not even React will reach that 😅 ... also I believe whenever that happens this code won't be needed or exist anymore, I am just a bit pragmatic here but no hard feelings with using current logic.